createdisk.sh

```diff
@@ -38,8 +38,13 @@ function create_crc_libvirt_sh {
 function create_qemu_image {
     local destDir=$1
 
-    sudo cp /var/lib/libvirt/images/${VM_PREFIX}-master-0 $destDir
-    sudo cp /var/lib/libvirt/images/${VM_PREFIX}-base $destDir
+    if [ -f /var/lib/libvirt/images/${VM_PREFIX}-master-0 ]; then
```

Comment:
Would be cleaner to check for libvirt storage pool existence, get the path from there, and use it. But given our use of sh to do all of this, the way you did is much easier/efficient.